pom.xml

```diff
@@ -145,6 +145,11 @@
             <artifactId>validation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.0</version>
```

Comment:
FindBugs itself allows for more granular suppression via XML filter files (http://findbugs.sourceforge.net/manual/filter.html), but I don't think Qulice currently supports this.